Gems/ROS2/Code/Source/VehicleDynamics/ModelLimits/SkidSteeringModelLimits.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "SkidSteeringModelLimits.h"
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace ROS2::VehicleDynamics
+{
+    void SkidSteeringModelLimits::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serialize = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serialize->Class<SkidSteeringModelLimits>()
+                ->Version(1)
+                ->Field("LinearLimit", &SkidSteeringModelLimits::m_linearLimit)
+                ->Field("AngularLimit", &SkidSteeringModelLimits::m_angularLimit);
+
+            if (AZ::EditContext* ec = serialize->GetEditContext())
+            {
+                ec->Class<SkidSteeringModelLimits>("Skid steering Model Limits", "Limitations of speed, steering angles and other values")
+                    ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                    ->Attribute(AZ::Edit::Attributes::Category, "ROS2")
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default,
+                        &SkidSteeringModelLimits::m_linearLimit,
+                        "Linear speed Limit",
+                        "Max linear speed (mps)")
```

Comment:
very minor, mps is too vague. 
```suggestion
                        "Max linear speed (meters/sec)")
```